src/LintingViewExtension/LinterView.xaml

```diff
@@ -0,0 +1,176 @@
+            </ControlTemplate>
+
+            <ControlTemplate x:Key="issueExpander"
+                             TargetType="{x:Type Expander}">
+                <Border>
+                    <DockPanel>
+                        <ToggleButton x:Name="ExpanderButton"
+                                      DockPanel.Dock="Top"
+                                      Template="{StaticResource expanderButton}"
+                                      Content="{TemplateBinding Header}"
+                                      IsChecked="{Binding Path=IsExpanded, 
+                      RelativeSource={RelativeSource TemplatedParent}}"
+                                      OverridesDefaultStyle="True"
+                                      Padding="1.5,0">
+                        </ToggleButton>
+                        <ContentPresenter x:Name="ExpanderContent"
+                                          Visibility="Collapsed"
+                                          DockPanel.Dock="Bottom"/>
+                    </DockPanel>
+                </Border>
+
+                <ControlTemplate.Triggers>
+                    <Trigger Property="IsExpanded"
+                             Value="True">
+                        <Setter TargetName="ExpanderContent"
+                                Property="Visibility"
+                                Value="Visible" />
+                    </Trigger>
+                </ControlTemplate.Triggers>
+            </ControlTemplate>
+        </ResourceDictionary>
+    </UserControl.Resources>
+
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="100" />
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+            <RowDefinition Height="50" />
+        </Grid.RowDefinitions>
+
+        <HeaderedContentControl Header="Graph Type"
+                                HeaderTemplate="{StaticResource HeaderItemControlHeaderTemplate}"
+                                Margin="16 10">
+            <Grid>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="*" />
+                    <ColumnDefinition Width="50" />
+                </Grid.ColumnDefinitions>
+                <ComboBox Grid.Column="0"
+                          Name="LinterSelector"
+                          ItemsSource="{Binding LinterManager.AvailableLinters}"
+                          SelectedItem="{Binding ActiveLinter, Mode=TwoWay}"
+                          DisplayMemberPath="Name"
+                          Style="{StaticResource SComboBox}"
+                          Height="30"
+                          Margin="0 15" />
+
+                <fa:ImageAwesome Icon="InfoCircle"
+                                 Grid.Column="1"
+                                 Foreground="#808080"
+                                 Width="20"
+                                 Height="20"
+                                 ToolTip="Some Tooltip"/>
```

Comment:
@nate-peters i forgot what we agreed should go into this `info` icon?

Right now its simply an icon with a tooltip that displays when hovering it.

![image](https://user-images.githubusercontent.com/13732445/115386800-caf19980-a1d1-11eb-96f8-a7c224e4ccb9.png)

Reply:
The idea was that icon would link to some documentation about Linting and Graph Rules in the new Dynamo Documentation Browser - @mjkkirschner and I talked about this yesterday and I can make a separate PR that adds the documentation in the expected places. Lets just leave the button here for now and I'll hook it up to the docs later. The tooltip should say "What is a Graph Type?"

Reply:
@nate-peters see this comment https://github.com/DynamoDS/Dynamo/pull/11634#issuecomment-828378420. For now i added a simple placeholder html file that will open when the button is pressed.